Code/Editor/CryEdit.cpp

```diff
@@ -3728,24 +3728,24 @@ void CCryEditApp::OnToolsPreferences()
 //////////////////////////////////////////////////////////////////////////
 void CCryEditApp::OnSwitchToDefaultCamera()
 {
-    CViewport* vp = GetIEditor()->GetViewManager()->GetSelectedViewport();
-    if (CRenderViewport* rvp = viewport_cast<CRenderViewport*>(vp))
-    {
-        rvp->SetDefaultCamera();
-    }
+    //CViewport* vp = GetIEditor()->GetViewManager()->GetSelectedViewport();
+    //if (CRenderViewport* rvp = viewport_cast<CRenderViewport*>(vp))
+    //{
+    //    rvp->SetDefaultCamera();
+    //}
 }
 
 //////////////////////////////////////////////////////////////////////////
-void CCryEditApp::OnUpdateSwitchToDefaultCamera(QAction* action)
+void CCryEditApp::OnUpdateSwitchToDefaultCamera([[maybe_unused]] QAction* action)
```

Comment:
It appears that now all of these View -> Viewport -> Switch Camera menu options are permanently disabled, so we should either remove them if they no longer apply, or make them work.

@NicholasVanSickle  @rainbj

Reply:
Yes, this change disables those options completely.

You can already do the same through camera component "Be this camera" and the "viewport camera selector". My reasoning is there really doesn't need to be a third way to do this (we personally do not use it), and it's part of legacy functionality, so I got rid of it.